src/util/xml_utils.js

```diff
@@ -63,7 +65,7 @@ function append_object(append, object) {
             // otherwise ignore the key and value altogether
             if (key[0] === '_') {
                 if (key === '_content') {
-                    append_object(append, val);
+                    append_object(append, val, repeat_array_tags, key);
```

Comment:
key here is the literal string "_content", but the parent tag name is probably still father (or mother, or better - parent).